package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The only GroupMe API library that isn't a million years old.",
   "main": "dist/index.js",
   "scripts": {
-    "prepare": "npx tsc",
+    "build": "npx tsc",
+    "prepare": "husky install",
```

Comment:
This `prepare` script overwrites the previous one, breaking all of our current CI.

Reply:
I can change the CI, and let this be the new prepare/build scripts

Reply:
are you able to add commits to this PR? that would be desirable

Reply:
I don't have write access to this repo